modules/katello_upload.py

```diff
@@ -68,7 +68,7 @@
     username: "admin"
     password: "changeme"
     server_url: "https://foreman.example.com"
-    file: "my.rpm"
+    src: "my.rpm"
```

Comment:
Explain a bit the change from `file` to `src`

Reply:
file is a built-in in Python, so using it as a variable is discouraged, as it shadows the built-in. and I've chosen src as that is the common name in other modules that involve acting on files or urls.